package.json

```diff
@@ -143,7 +143,7 @@
     "release-it-lerna-changelog": "^2.3.0"
   },
   "peerDependencies": {
-    "ember-data": "2.x - 3.x",
+    "ember-data": "3.x",
```

Comment:
I think we should do `>= 3.0.0`, but not sure. I would want it to work with 4.x too later potentially.